include/nbl/builtin/hlsl/blit/normalization.hlsl

```diff
@@ -0,0 +1,108 @@
+// Copyright (C) 2023 - DevSH Graphics Programming Sp. z O.O.
+// This file is part of the "Nabla Engine".
+// For conditions of distribution and use, see copyright notice in nabla.h
+#ifndef _NBL_BUILTIN_HLSL_BLIT_NORMALIZATION_INCLUDED_
+#define _NBL_BUILTIN_HLSL_BLIT_NORMALIZATION_INCLUDED_
+
+#include <nbl/builtin/hlsl/workgroup/arithmetic.hlsl>
+#include <nbl/builtin/hlsl/blit/parameters.hlsl>
+#include <nbl/builtin/hlsl/glsl_compat/core.hlsl>
+
+namespace nbl
+{
+namespace hlsl
+{
+namespace blit
+{
+
+
+//! WARNING: ONLY WORKS FOR `dividendMsb<=2^23` DUE TO FP32 ABUSE !!!
+uint32_t integerDivide_64_32_32(in uint32_t dividendMsb, in uint32_t dividendLsb, in uint32_t divisor)
+{
+	const float msbRatio = float(dividendMsb) / float(divisor);
+	const uint32_t quotient = uint32_t((msbRatio * (~0u)) + msbRatio) + dividendLsb / divisor;
+	return quotient;
+}
+
+struct OpAdd
+{
+	uint operator()(uint v1, uint v2) { return v1 + v2; }
+};
```

Comment:
we have this in `binops` , lets wait on the normalization review till #519 gets merged